README.md

```diff
@@ -52,6 +52,7 @@ To build for standalone use in the browser, install `browserify` and check [run-
     - [`vm.generateGenesis(cb)`](#vmgenerategenesiscb)
   - [`VM` debugging hooks](#vm-debugging-hooks)
     - [`vm.onStep`](#vmonstep)
+    - [`newContract` event](#newcontract)
```

Comment:
Any reasons for the inconsistent naming in regards to the ``vm.onStep`` event description title?

Reply:
I guess I've disagreed with `vm.onStep` as it's not a function, so I named it something I felt was more intuitive without changing more than was my realm.

I can make it however you want. `vm.onNewContractEvent` to me is just incorrect.